<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>715</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="igrac6_D">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>300</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="igrac9_L">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>300</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="maxVisibleItems">
    <number>30</number>
   </property>
   <property name="duplicatesEnabled">
    <bool>true</bool>
   </property>
   <property name="modelColumn">
    <number>0</number>
   </property>
  </widget>
  <widget class="QComboBox" name="igrac8_D">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>60</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="igrac10_L">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>540</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="igrac5_L">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>60</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="igrac6_L">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>300</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="igrac10_D">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>540</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="igrac7_L">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>540</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="igrac7_D">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>540</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="golman_L">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="igrac8_L">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>60</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="golman_D">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>250</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="igrac5_D">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>60</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="igrac9_D">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>300</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="igrac1_L">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="igrac2_L">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>210</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="igrac3_L">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>390</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="igrac4_L">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>550</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>131</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="slike.qrc">:/slike/slike/imageedit_5_4386524822.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>250</y>
     <width>131</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="slike.qrc">:/slike/slike/imageedit_5_4386524822.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>420</y>
     <width>131</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="slike.qrc">:/slike/slike/imageedit_5_4386524822.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>590</y>
     <width>131</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="slike.qrc">:/slike/slike/imageedit_5_4386524822.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>290</y>
     <width>131</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="slike.qrc">:/slike/slike/imageedit_5_4386524822.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>330</y>
     <width>131</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="slike.qrc">:/slike/slike/imageedit_5_4386524822.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>330</y>
     <width>131</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="slike.qrc">:/slike/slike/imageedit_5_4386524822.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>90</y>
     <width>131</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="slike.qrc">:/slike/slike/imageedit_5_4386524822.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>90</y>
     <width>131</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="slike.qrc">:/slike/slike/imageedit_5_4386524822.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>570</y>
     <width>131</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="slike.qrc">:/slike/slike/imageedit_5_4386524822.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>570</y>
     <width>131</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="slike.qrc">:/slike/slike/imageedit_5_4386524822.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>280</y>
     <width>131</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="slike.qrc">:/slike/slike/imageedit_7_8437819462.png</pixmap>
   </property>
  </widget>
  <widget class="QComboBox" name="igrac4_D">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>550</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="igrac3_D">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>380</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="igrac2_D">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>210</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="igrac1_D">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>40</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>410</y>
     <width>131</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="slike.qrc">:/slike/slike/imageedit_7_8437819462.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>70</y>
     <width>131</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="slike.qrc">:/slike/slike/imageedit_7_8437819462.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>240</y>
     <width>131</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="slike.qrc">:/slike/slike/imageedit_7_8437819462.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>580</y>
     <width>131</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="slike.qrc">:/slike/slike/imageedit_7_8437819462.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>330</y>
     <width>131</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="slike.qrc">:/slike/slike/imageedit_7_8437819462.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>330</y>
     <width>131</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="slike.qrc">:/slike/slike/imageedit_7_8437819462.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>570</y>
     <width>131</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="slike.qrc">:/slike/slike/imageedit_7_8437819462.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>570</y>
     <width>131</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="slike.qrc">:/slike/slike/imageedit_7_8437819462.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>90</y>
     <width>131</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="slike.qrc">:/slike/slike/imageedit_7_8437819462.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_22">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>90</y>
     <width>131</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="slike.qrc">:/slike/slike/imageedit_7_8437819462.png</pixmap>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_L_igrac1">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>100</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2_L_igrac2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>280</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3_L_golman">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>320</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_4_L_igrac5">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>120</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_5_L_igrac8">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>120</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_6_L_igrac6">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>360</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_7_L_igrac9">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>360</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_8_L_igrac3">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>450</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_9_L_igrac7">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>600</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_10_L_igrac4">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>620</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_11_L_igrac10">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>600</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_12_D_igrac10">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>610</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_13_D_igrac7">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>610</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_14_D_igrac4">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>620</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_15_D_golman">
   <property name="geometry">
    <rect>
     <x>1140</x>
     <y>320</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_16_D_igrac3">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>450</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_17_D_igrac2">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>280</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_18_D_igrac6">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>370</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_19_D_igrac9">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>370</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_20_D_igrac1">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>110</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_21_D_igrac5">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>130</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_22_D_igrac8">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>130</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="snimi">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>660</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>SNIMI</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Le_L_golman">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>360</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Le_L_igrac1">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>140</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Le_L_igrac2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>320</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Le_L_igrac3">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>490</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Le_L_igrac4">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>660</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Le_L_igrac5">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>160</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Le_L_igrac6">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>400</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Le_L_igrac7">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>640</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Le_L_igrac8">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>160</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Le_L_igrac9">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>400</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Le_L_igrac10">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>640</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Le_D_golman">
   <property name="geometry">
    <rect>
     <x>1130</x>
     <y>360</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Le_D_igrac1">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>150</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Le_D_igrac2">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>320</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Le_D_igrac3">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>490</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Le_D_igrac4">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>660</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Le_D_igrac5">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>170</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Le_D_igrac6">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>410</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Le_D_igrac7">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>650</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Le_D_igrac8">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>170</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Le_D_igrac9">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>410</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Le_D_igrac10">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>650</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="TEST">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>400</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="slike.qrc"/>
 </resources>
 <connections/>
</ui>
